File: app/src/main/java/dev/ragnarok/filegallery/model/LocalServerSettings.java
package dev.ragnarok.filegallery.model;

import com.google.gson.annotations.SerializedName;

public class LocalServerSettings {
    @SerializedName("url")
    public String url;

    @SerializedName("password")
    public String password;

    @SerializedName("enabled")
    public boolean enabled;

    @SerializedName("enabled_audio_local_sync")
    public boolean enabled_audio_local_sync;
}

File: app/src/main/kotlin/dev/ragnarok/filegallery/model/LocalServerSettings.kt
package dev.ragnarok.filegallery.model

import com.google.gson.annotations.SerializedName

class LocalServerSettings {
    @SerializedName("url")
    var url: String? = null

    @SerializedName("password")
    var password: String? = null

    @SerializedName("enabled")
    var enabled = false

    @SerializedName("enabled_audio_local_sync")
    var enabled_audio_local_sync = false
}